src/headers/agent_op.h

```diff
@@ -84,12 +84,55 @@ int auth_connect();
 // Close socket if valid.
 int auth_close(int sock);
 
-// Send a local agent add request.
-int w_request_agent_add_local(int sock, char *id, const char *name, const char *ip, const char * groups, const char *key, const int force, const int json_format, const char *agent_id, int exit_on_error);
+/**
+ * @brief Send a local agent add request.
+ * @param sock Socket where the request connection will be done.
+ * @param id ID of the newly generated key.
+ * @param name Name of the agent to request the new key.
+ * @param ip IP of the agent to request the new key.
+ * @param groups Groups list of the agent to request the new key.
+ * @param key KEY of the newly generated key.
+ * @param force Force option to be used during the registration. -1 means disabled. 0 or a positive value means enabled.
+ * @param json_format Flag to identify if the response should be printed in json format.
```

Comment:
I would use capital letters in the `JSON` string.

Reply:
Done